integrations/tensorflow/bindings/python/pyiree/tools/tflite/__init__.py

```diff
@@ -0,0 +1,29 @@
+# Lint-as: python3
+"""TFLite tools."""
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+import os
+import platform
+
+
+def get_tool(exe_name: str) -> Optional[str]:
```

Comment:
worth sharing? (duped here and it the xla importer at least)

Reply:
No - there isn't any real place to share it *to*. This is a top-level module, deployed as part of its own package. I'd need to create a separate "tools" package of some kind. It's not too much, so I just duped it.